versions.yaml

```diff
@@ -1,14 +1,17 @@
-release: 0.0.1-4554
+release: 0.0.1-5318
 images:
   - name: controller-dp
-    path: ghcr.io/open-traffic-generator/ixia-c-controller
-    tag: 0.0.1-4554
+    path: ghcr.io/open-traffic-generator/keng-controller
+    tag: 0.0.1-5318
   - name: controller-cpdp
-    path: ghcr.io/open-traffic-generator/licensed/ixia-c-controller
-    tag: 0.0.1-4554
+    path: ghcr.io/open-traffic-generator/keng-controller
+    tag: 0.0.1-5318
+  - name: keng-license-server
+    path: ghcr.io/open-traffic-generator/licensed/keng-license-server
+    tag: 0.0.1-24
```

Comment:
@biplamal it should contain `latest` in all places including CI of keng-controller/ixops